tests/pytests/test.py

```diff
@@ -1230,7 +1230,7 @@ def testInKeys(env):
     with env.assertResponseError():
         env.cmd('ft.search', 'idx', 'hello', 'INKEYS', -1)
     with env.assertResponseError():
-        env.cmd('ft.search', 'idx', 'hello', 'inkeys', 4, 'foo')
+        env.cmd('ft.search', 'idx', 'hello', 'inkeys', 5, 'foo')
```

Comment:
why is this change needed?

Reply:
We have a not-ideal validation mechanism, that does not know to distinguish between the user and coordinator arguments in a shard command.
That yields this weird situation that we have today, and should be solved by performing the validations once in the coordinator I think.
Till then, we have a mixed state and this is just a workaround for that

Reply:
I didn't understand. If we search for `'ft.search', 'idx', 'hello', 'inkeys', 4, 'foo'` we don't fail?

Reply:
No. Since we add arguments to the shard command, from the coordinator.

Reply:
I'll add it sooner so we won't have this issue on `FT.SEARCH`.

Reply:
This sounds like a bug, no? What do we get if not an error? Does it consume the prefixes as the in-keys? Shouldn’t we fail in this case?